gateway/shellhub.conf

```diff
@@ -392,12 +392,13 @@ server {
     }
 }
 
+{{- PUBLIC_URL_DOMAIN := or (env.Getenv "SHELLHUB_PUBLIC_URL_DOMAIN") (env.Getenv "SHELLHUB_DOMAIN") }}
 server {
    listen 80;
-   server_name ~^(?<namespace>.+)\.(?<device>.+)\..+$;
+   server_name ~^(?<namespace>.+)\.(?<device>.+)\.{{ $PUBLIC_URL_DOMAIN }}$;
```

Comment:
Is the same condition as using `SHELLHUB_DOMAIN` as default isn't required here?